custom_components/mawaqit/manifest.json

```diff
@@ -1,9 +1,9 @@
 {
   "domain": "mawaqit_prayer_times",
   "name": "Mawaqit Prayer Times",
-  "documentation": "https://github.com/mawaqit/home-assistant",
-  "requirements": ["mawaqit_times_calculator>=0.0.4","pymysql"],
-  "codeowners": ["@mawaqit"],
+  "documentation": "https://www.home-assistant.io/integrations/mawaqit_prayer_times",
```

Comment:
this url is ko

Reply:
we can use the github one waiting for having the integration officially adopted by home assistant.
the requirements should be empty. I will remove them.
the mawaqit python library is rewritten from scratch. it is proposed in the file mawaqit.py.
can you submit it to pypi from your side?